modules/ROOT/pages/veevavault/veevavault-connector-studio.adoc

```diff
@@ -0,0 +1,177 @@
+
+== Configure the Connector
+
+To use VeevaVault Connector in your Mule app, configure a global VeevaVault element that can be used by all VeevaVault Connector operations in the application.
+
+. To create a global element for the connector, click the *Global Elements* tab at the bottom of the canvas and click *Create*.
++
+. In the *Choose Global Type* window, expand *Connector Configuration*, click *Veeva Vault Config*, and click *OK*.
++
+. In the Veeva Vault Config window, in the *General* tab, enter the configuration values:
++
+image::veevavault/veevavault-connector-basic-authentication-general.png[]
++
+[%header%autowidth.spread]
+|===
+|*Field* |*Description*
+|*Name* |Enter a name for this configuration to reference it later.
+|*Vault URL* |Enter the Veeva Vault instance URL.
+|*Username* |Enter the Veeva Vault instance username.
+|*Password* |Enter the password that corresponds to the username.
+|*Version* |Enter the Veeva Vault instance version.
+|*Client Id* |Enter the client ID to send with each request in the header for tracking purposes, in the format: `\{organisation}-{company\|team}`. +
+The client ID is prefixed with "mulesoft" and has a suffix with "client-vaultconnector" internally, for example: `mulesoft-abcpharma-clinicalprod-client-vaultconnector`. 
+|===
++
+In the previous image, the placeholder values refer to a configuration file placed in the folder (`src/main/resources`) of your project. You can either hardcode your credentials into the global configuration properties or reference a configuration file that contains these values. +
+For simpler maintenance and better reusability of your project, using a configuration file is recommended. Keeping these values in a separate file is useful if you need to deploy to different environments, such as production, development, and QA, where your access credentials differ. +
+The following is an example snippet of the `configuration.yaml` file under the `src/main/resources/config` folder:
++
+[source,yaml,linenums]
+----
+	#Veeva Vault details
+	veeva:
+		vaultURL: "https://xxxx-xxxx.veevavault.com"
+		username: "someone@example.com"
+		password: "xxxxxxx"
+		version : "v19.1"
+		clientId: "mulesoft-abcpharma-clinicalprod-client-vaultconnector"
+----
++
+The connection configuration performs basic authentication, which accepts the required Vault URL, username, and password parameters to connect to the Veeva Vault instance to retrieve the session. The retrieved session is used for each operation configured in the Mule application for sending requests.
+. In the Veeva Vault Config window, *Advanced* tab, configure the burst API limit.
++
+image::veevavault/veevavault-connector-basic-authentication-advanced.png[]
++
+*Burst API Limit Profile*
++
+The burst API limit profile configuration to use if the specified burst API limit of the Veeva Vault API reaches the threshold of the configured value or is exceeded.
```

Comment:
This not being a complete sentence threw me off for a bit, but then I realized that this was like a definition list, in which case you want to remove the period.
I have edited for this in all the peer texts. Whatever you decide to do, be sure they are all parallel.

Reply:
```suggestion
The burst API limit profile configuration to use if the specified burst API limit of the Veeva Vault API reaches the threshold of the configured value or is exceeded
```